tools/proteore_build_protein_interaction_maps/build_protein_interaction_maps.xml

```diff
@@ -0,0 +1,311 @@
+    1,10549,"A1BG","PRDX4","Negative Genetic","genetic",21988832,"NA","NA"
+    1,9923,"A1BG","ZBTB40","Affinity Capture-MS","physical",28514442,0.99977983,"NA"  
+
+
+**"Interaction Score"**: a positive for negative value recorded by the original publication depicting P-Values, Confidence Score, SGA Score, etc. Will be “NA” if no score is reported.
+
+"Nodes" output file (example):
+
+.. csv-table:: Nodes file (if BioGRID database selected - simulated data)
+   :header: "Entrez gene ID","Official Symbol Interactor","Present in user input ids","ID present in Biogrid Human","Pathway"
+   
+	"1","A1BG","True","True","Platelet degranulation ;Neutrophil degranulation"
+	"10","NAT2","False","True","Acetylation"
+	"12","SERPINA3","True","False","NA"
+
+
+
+- **"Present in user input ids"**: a boolean value (i.e. True/False) telling you whether the ID was present in your initial list ("True") or not ("False"); a "False" indicates that this ID is not present in your list but has been reported in BioGRID.    
+- **"ID present in Biogrid Human"**: a boolean value (i.e. True/False) telling you whether the ID was present in the BioGRID release you selected ("True") or not ("False"); a "False" value indicates that one ID from your list is not reported in the BioGRID release used.  
+
+These 2 files can be directly imported into a visualization software (such as Cytoscape - https://cytoscape.org/download.html) for further exploration and analysis of the newly created biological network.
+
+-----
+
+.. class:: infomark
+
+**Data source (release date)**
+
+This tool uses the following public ressources (for more details please check: http://www.proteore.org/static/data_source.html)
+
+- **BioGRID**:
+
+Data were downloaded from BioGrid: https://downloads.thebiogrid.org/BioGRID/
+
+Installation date: 
+
+06/02/2020
+
+01/03/2019
+
+
+- **Bioplex**: Version 2.0 
+
+BioPlex_interactionList_v4a.tsv: http://bioplex.hms.harvard.edu/data/BioPlex_interactionList_v4a.tsv
+
+- **Human protein complex Map (Hu.map)**:
+
+nodeTable.txt: http://proteincomplexes.org/static/downloads/nodeTable.txt
+
+pairsWprob: http://proteincomplexes.org/static/downloads/pairsWprob.txt
+
+Mapping files linking the source database identifier (Entrez gene ID and Uniprot Accession Number) to the Reactome pathways are based on 2018-12-07 version, NCBI2Reactome.txt and UniProt2Reactome.txt (from https://www.reactome.org/download-data).
+
+-----
+
+.. class:: infomark
+
+**Galaxy integration**
+
+David Christiany, Lisa Perus, Florence Combes, Yves Vandenbrouck - CEA, INSERM, CNRS, Grenoble-Alpes University, BIG Institute, FR
+
+Sandra Dérozier, Olivier Rué, Valentin Loux - INRA, Paris-Saclay University, MAIAGE Unit, Migale Bioinformatics platform, FR
+
+This work has been partially funded through the French National Agency for Research (ANR) IFB project.
+
+Help: contact@proteore.org for any questions or concerns about the Galaxy implementation of this tool.
+    ]]></help>
+    <citations>
```

Comment:
can you add a bibtex style citations and cite the authors that you have enough. Then the "citations" are more structured and not part of the help.